lib/aggregators/log-aggregator.js

```diff
@@ -39,6 +39,7 @@ class LogAggregator extends EventAggregator {
   }
 
   add(logLine) {
+    logLine = typeof logLine === 'string' ? JSON.parse(logLine) : logLine
```

Comment:
ok talked this out with team.  this will happen on every log line added to aggregator regardless of sample size and priority. It's probably best to de-serialize in `_toPayloadSync`